wagtail/admin/templates/wagtailadmin/generic/form.html

```diff
@@ -16,7 +16,7 @@ <h2 class="w-header__title" id="header-title">
 
 {% block main_content %}
     {% block before_form %}{% endblock %}
-    <form action="{{ action_url }}" method="POST" novalidate{% if form.is_multipart %} enctype="multipart/form-data"{% endif %}>
+    <form action="{{ action_url }}" method="POST" novalidate{% if form.is_multipart %} enctype="multipart/form-data"{% endif %} data-edit-form>
```

Comment:
This attribute is used by various components (preview panel, scheduled publishing dialog) when they need to refer to the main "editor" form. The "edit" here doesn't mean it's exclusive for edit views – it's also used for the create view.

We probably should rename this in the future, but that's not important for now.

Reply:
Yeah this confused me also when reviewing the unsaved confirmation controller stuff.

This too will be something we can rename or rethink as we move more things over to controller attributes on the relevant forms.

However, it may always be useful to have some kind of central attribute on our primary forms that other selectors can hook into.